src/pilot/install-director.sh

```diff
@@ -39,17 +39,39 @@ while true ; do
                 subscription_manager_pass=$2 ; shift 2 ;;
         --sm_pool)
                 subcription_manager_poolid=$2; shift 2 ;; 
+        --satellite_hostname)
+                satellite_hostname=$2; shift 2;;
+        --satellite_org)
+                satellite_org=$2; shift 2;;
+        --satellite_key)
+                satellite_key=$2; shift 2;;
+        --containers_prefix)
+                containers_prefix=$2; shift 2;;
         --proxy)
                 proxy=$2; shift 2 ;;
         --nodes_pwd)
                 overcloud_nodes_pwd=$2; shift 2 ;;
         --) shift ; break ;;
-        *) echo "$USAGE" ; exit 1 ;;
+        *) echo -e "$USAGE" ; exit 1 ;;
     esac
 done
 
-if [ -z "${dns_ip}" ] || [ -z "${subscription_manager_user}" ] || [ -z "${subscription_manager_pass}" ]; then
-    echo $USAGE
+
+if [ ! -z "${satellite_hostname}" ]; then
+   
+    if [ -z "${dns_ip}" ] || [ -z "${satellite_hostname}" ] || [ -z "${satellite_org}" ] || [ -z ${satellite_key} ] ; then
+        echo -e "$USAGE"
```

Comment:
It looks to me like the usage statement echo'ed above is not correct.

Reply:
Ah - never mind - i see that USAGE contains the usage for both satellite and CDN